controllers/managedocs_controller_test.go

```diff
@@ -373,13 +402,37 @@ var _ = Describe("ManagedOCS controller", func() {
 				By("Creating a storagecluster resource")
 				utils.WaitForResource(k8sClient, ctx, scTemplate.DeepCopy(), timeout, interval)
 
-				By("Creating a prometous resource")
+				By("Creating a prometheus resource")
 				utils.WaitForResource(k8sClient, ctx, promTemplate.DeepCopy(), timeout, interval)
 
 				By("Creating an alertmanager resource")
 				utils.WaitForResource(k8sClient, ctx, amTemplate.DeepCopy(), timeout, interval)
 			})
 		})
+		When("there is no rook-ceph-operator-config ConfigMap", func() {
+			It("should not create reconciled resources", func() {
```

Comment:
Me reasoning for enforcing this behavior is as follows: If we can't set the limits on the resources we shouldn't run the resources. Let me know if this is unreasonable and I will change it.